lib/osf-components/addon/components/validated-input/text/styles.scss

```diff
@@ -1,5 +1,14 @@
-.PrefixedInputBox {
-    display: flex !important;
+:global(.form-control).PrefixedInputBox {
```

Comment:
This is a really nice example of using CSS modules. Do you know if using the global affects any other classes for form-control elements on the page? I read CSS modules are locally instantiated, does this function similar to using local-class in the template file?

Reply:
This rule `:global(.form-control).PrefixedInputBox` will apply to any hbs element with 
`<tag class='form-control' local-class='PrefixedInputBox' >`, or once it's rendered in the browser, any html element with `<tag class='form-control _PrefixedInputBox_123someID123`. Since I stipulate that this rule applies only to elements with both the global-class and this specific local-class, other `.form-control` elements are unaffected by this.

A "leaky" implementation of this css rule could look something like this:
```
:global(.form-control) {
    color: blue;

    &.PrefixedInputBox {
        border: solid 1px green;
    }
}
```
In this example, I'm setting a rule for elements with the `.form-control` global class and the `.PrefixedInputBox` local-class like the code in this PR, but _all_ elements that have the `.form-control` global-class are being impacted by the first `color` rule as well